### ISSUE TITLE
print can not work normally

### ISSUE THREAD

**Issue by swoec** [spyder-ide/spyder#6946]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "/home/alex/anaconda2/lib/python2.7/site-packages/spyder_profiler/widgets/profilergui.py", line 264, in <lambda>
    self.finished(ec, es))
  File "/home/alex/anaconda2/lib/python2.7/site-packages/spyder_profiler/widgets/profilergui.py", line 329, in finished
    self.show_data(justanalyzed=True)
  File "/home/alex/anaconda2/lib/python2.7/site-packages/spyder_profiler/widgets/profilergui.py", line 350, in show_data
    self.datatree.load_data(self.DATAPATH)
  File "/home/alex/anaconda2/lib/python2.7/site-packages/spyder_profiler/widgets/profilergui.py", line 462, in load_data
    stats_indi = [pstats.Stats(profdatafile),]
  File "/home/alex/anaconda2/lib/python2.7/pstats.py", line 81, in __init__
    self.init(arg)
  File "/home/alex/anaconda2/lib/python2.7/pstats.py", line 95, in init
    self.load_stats(arg)
  File "/home/alex/anaconda2/lib/python2.7/pstats.py", line 109, in load_stats
    f = open(arg, 'rb')
IOError: [Errno 2] No such file or directory: u'/home/alex/.config/spyder/profiler.results'

## Version and main components

* Spyder Version: 3.2.6 
* Python Version: 2.7.14
* Qt Versions: 5.6.2, PyQt5 5.6 on Linux

## Dependencies
```
pyflakes >=0.5.0  :  1.6.0 (OK)
pycodestyle >=2.3 :  2.3.1 (OK)
pygments >=2.0    :  2.2.0 (OK)
pandas >=0.13.1   :  0.22.0 (OK)
numpy >=1.7       :  1.14.0 (OK)
sphinx >=0.6.6    :  1.6.6 (OK)
rope >=0.9.4      :  0.10.7 (OK)
jedi >=0.9.0      :  0.11.1 (OK)
psutil >=0.3      :  5.4.3 (OK)
nbconvert >=4.0   :  5.3.1 (OK)
sympy >=0.7.3     :  1.1.1 (OK)
cython >=0.21     :  0.27.3 (OK)
qtconsole >=4.2.0 :  4.3.1 (OK)
IPython >=4.0;<6.0:  5.4.1 (OK)
pylint >=0.25     :  1.8.2 (OK)

```

**Comment by ccordoba12** [comment]
This is fixed in our latest version (3.2.8). Please update by opening the Anaconda Prompt and running there

    conda update spyder

**Comment by CAM-Gerlach** [comment]
Specifically, its a duplicate of #6220 fixed in Spyder 3.2.7.
